Projects/CMakeLists.txt

```diff
@@ -262,14 +258,8 @@ IF(LINUX)
     FIND_PACKAGE(OpenAL REQUIRED)
 ENDIF()
 
-#Build static libraries only for FFTW
+#Build static libraries only for subprojects
 SET(BUILD_SHARED_LIBS OFF CACHE BOOL "" FORCE)
-#Enable floating point for FFTW
-SET(ENABLE_FLOAT ON CACHE BOOL "" FORCE)
```

Comment:
Does removing these not have an impact?

Reply:
No. It was used for a now non-existant library (FFTW)